src/style.css

```diff
@@ -1,3 +1,73 @@
-body {
+html{
+  font-family: Roboto, sans-serif;
+
+}
+
+body{
+  --main-color: black;
+  --active-color: #00ACDC;
+
+  background-color: antiquewhite;
   margin: 0;
+  padding: 0;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.header{
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)